aptos-move/move-examples/token_objects/sources/token.move

```diff
@@ -72,22 +70,25 @@ module token_objects::token {
     /// Creates a new token object and returns the ConstructorRef for additional specialization.
     public fun create(
         creator: &signer,
-        collection: String,
+        collection_name: String,
         description: String,
         name: String,
         royalty: Option<Royalty>,
         uri: String,
     ): ConstructorRef {
         let creator_address = signer::address_of(creator);
-        let seed = create_token_seed(&collection, &name);
+        let seed = create_token_seed(&collection_name, &name);
+
+        let collection_addr = collection::create_collection_address(&creator_address, &collection_name);
+        let collection = object::address_to_object<Collection>(collection_addr);
+        let id = collection::increment_supply(&collection);
```

Comment:
Do we really want this id? Also can we use aggergator for collection's supply to allow parallel minting?

Reply:
What later circumstances? Also why is the supply not aggregator based? It's better to use aggregator since the beginning esp as this is going into the framework rather than having to do a migration later on

Reply:
IIUC, the id and aggregator are sort of mutually exclusive. If we have aggregator, we shouldn't have id, vice versa.